src/test/java/com/rackspace/ceres/app/web/WriteControllerTest.java

```diff
@@ -49,26 +49,6 @@
   @Captor
   ArgumentCaptor<Flux<Tuple2<String, Metric>>> metrics;
 
-  @Test
-  public void testPutMetrics() {
```

Comment:
Why remove this test? Its still valid even without the default tenant right? We just need to define a tenant to pass in.

Reply:
Other test case `testPutMetrics_WithParams` covers the same scenario. This test was checking for default tenant only.